Assets/Scripts/ConnectionManagement/ConnectionState/ClientReconnectingState.cs

```diff
@@ -81,19 +84,33 @@ IEnumerator ReconnectCoroutine()
             m_NbAttempts++;
             if (!string.IsNullOrEmpty(m_LobbyCode))
             {
-                var leavingLobby = m_LobbyServiceFacade.EndTracking();
-                yield return new WaitUntil(() => leavingLobby.IsCompleted);
-                var joiningLobby = m_LobbyServiceFacade.TryJoinLobbyAsync("", m_LobbyCode);
-                yield return new WaitUntil(() => joiningLobby.IsCompleted);
-                if (joiningLobby.Result.Success)
+                // Attempting to reconnect to lobby
```

Comment:
```suggestion
                // Attempting to reconnect to lobby
                // Timeout configs have been set in Lobby's dashboard
```

Reply:
I'm not sure I understand what this comment aims to convey

Reply:
This is mostly to show users where timeouts are set for this, since we can't point anywhere in code

Reply:
I see, but the timeout set in the dashboard doesn't come into play here specifically. It is the amount of time between when a client is marked as being disconnected from a Relay server and when the Lobby service removes them from the Lobby. I'll write a more detailed comment to explain it